File: extensions/js/replay_gain.js
export function resolve_album_tag_replaygain_album_gain(ctx) {
    return String(ctx.source.REPLAYGAIN_ALBUM_GAIN || "");
}

export function resolve_album_tag_replaygain_album_peak(ctx) {
    return String(ctx.source.REPLAYGAIN_ALBUM_PEAK || "");
}

export function resolve_track_tag_replaygain_track_gain(ctx) {
    return String(ctx.source.REPLAYGAIN_TRACK_GAIN || "");
}

export function resolve_track_tag_replaygain_track_peak(ctx) {
    return String(ctx.source.REPLAYGAIN_TRACK_PEAK || "");
}

File: extensions/python/functions/replay_gain.py
def resolve_album_tag_replaygain_album_gain(ctx):
    return str(ctx["source"].get("REPLAYGAIN_ALBUM_GAIN", ""))

def resolve_album_tag_replaygain_album_peak(ctx):
    return str(ctx["source"].get("REPLAYGAIN_ALBUM_PEAK", ""))

def resolve_track_tag_replaygain_track_gain(ctx):
    return str(ctx["source"].get("REPLAYGAIN_TRACK_GAIN", ""))

def resolve_track_tag_replaygain_track_peak(ctx):
    return str(ctx["source"].get("REPLAYGAIN_TRACK_PEAK", ""))
